client/src/pages/private/ParticipantEOI.js

```diff
@@ -296,7 +297,7 @@ export default () => {
                       color='primary'
                       onClick={() => setOpenWithdraw(true)}
                     >
-                      Withdraw From the Program
+                      Withdraw PEOI
```

Comment:
Don't you think PEOI is a bit technical term. What about `Withdraw Interest`?

Reply:
Fair comment, I just copied the ticket, want to bring this up to the channel?

Reply:
Yes, do it.